RxSwift/Subjects/Variable.swift

```diff
@@ -10,7 +10,7 @@
 ///
 /// Unlike `BehaviorSubject` it can't terminate with error, and when variable is deallocated
 /// it will complete it's observable sequence (`asObservable`).
-public final class Variable<Element> {
+public final class Variable<Element>: ObservableConvertibleType {
```

Comment:
Thanks for the discussion.
Finally, I can understand that why `Variable` doesn't conform to `ObservableConvertibleType`.

@inamiy 
About `ObserverType`, I think that is not a real problem. For example, `UIBindingObserver` has a same implement that `func bindTo(_ variable: Variable<E>) -> Disposable`'s one.

If my understanding is correct, adding `ObserverConvertibleType` may help us. 
I think it is effective not only `Variable`.